From 1e145dc3df40a3d4a4b0130c974bf2d2aaff215d Mon Sep 17 00:00:00 2001
From: "qipeng.zha" <qipeng.zha@intel.com>
Date: Mon, 8 Dec 2014 19:38:55 +0800
Subject: [PATCH 1/1] Touch: decrease printk level for zero message

Touch firmware will put touch events into T100 object
buffer and trigger interrupt for each event.
There exist host driver read out all events from T100
object in the first ISR and touch firmware still trigger
interrupt for other events.
So driver will see such warning of "zero message" in ISR.

Decrease printk level of this warning to avoid boot delay.

Change-Id: I760f6e2d9907c14881315af0bbb2462897d8db53
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-11255
Signed-off-by: qipeng.zha <qipeng.zha@intel.com>
---
 drivers/input/touchscreen/atmel_mxt_ts.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/atmel_mxt_ts.c b/drivers/input/touchscreen/atmel_mxt_ts.c
index 4abbc2f..8e10b9a 100644
--- a/drivers/input/touchscreen/atmel_mxt_ts.c
+++ b/drivers/input/touchscreen/atmel_mxt_ts.c
@@ -1275,7 +1275,7 @@ static irqreturn_t mxt_process_messages_t44(struct mxt_data *data)
 	count = data->msg_buf[0];
 
 	if (count == 0) {
-		dev_warn(dev, "Interrupt triggered but zero messages\n");
+		dev_dbg(dev, "Interrupt triggered but zero messages\n");
 		return IRQ_NONE;
 	} else if (count > data->max_reportid) {
 		dev_err(dev, "T44 count %d exceeded max report id\n", count);
-- 
1.9.1

